addons/web/static/lib/hoot/tests/hoot-dom/events.test.js

```diff
@@ -1139,6 +1139,46 @@ describe(parseUrl(import.meta.url), () => {
         ]);
     });
 
+    test("edit with dirty value and blur", async () => {
+        await mountForTest(/* xml */ `
+            <input type="text" />
+            <button>Diversion</button>
+        `);
+        await click("input");
+        await edit("test value");
+
+        monitorEvents("input");
+
+        await click("button");
+
+        expect.verifySteps([
+            "pointermove:0@input",
+            "mousemove:0@input",
+            "pointerout:0@input",
+            "mouseout:0@input",
+            "pointerleave:0@input",
+            "mouseleave:0@input",
+            "change@input",
```

Comment:
I suppose this is handled elsewhere, but would not that be more robust if the events on the button were monitored as well ?

Reply:
@jum-odoo